generator/parser/codemodel.cpp

```diff
@@ -618,6 +626,10 @@ bool _FunctionModelItem::isSimilar(FunctionModelItem other) const
   if (isConstant() != other->isConstant())
     return false;
 
+  // ### this may be incorrect (note that isVolatile is not checked!)
+  if (isConstexpr() != other->isConstexpr())
+    return false;
+
```

Comment:
I think this is not needed, as `constexpr` functions cannot be overloaded.

Reply:
Removed